gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/source/HiveSource.java

```diff
@@ -515,8 +517,10 @@ public static void setLineageInfo(ConvertibleHiveDataset convertibleHiveDataset,
       String destTable = conversionConfigForFormat.get().getDestinationDbName() + "." + conversionConfigForFormat.get()
           .getDestinationTableName();
       DatasetDescriptor dest = new DatasetDescriptor(DatasetConstants.PLATFORM_HIVE, destTable);
-      Path destPath = new Path(conversionConfigForFormat.get().getDestinationDataPath());
-      dest.addMetadata(DatasetConstants.FS_URI, destPath.getFileSystem(new Configuration()).getUri().toString());
+      FileSystem destFs = sourceFs;
```

Comment:
Why is `destFs = sourceFs`? In general, a source can be ELT'ed to different destinations.